docs/deployment/cloud-scanning.md

```diff
@@ -0,0 +1,182 @@
+---
+slug: cloud-scanning
+title: Add repositories to Semgrep in bulk (beta)
+hide_title: true
+description: Semgrep cloud scanning provides an alternative to CI-based workflows. It enables you to add repositories to your Semgrep org in bulk without changing your CI workflows.
+tags:
+  - Deployment
+  - Semgrep AppSec Platform
+---
+
+# Add repositories to Semgrep in bulk
+
+**Semgrep cloud scanning** enables you to add repositories to your Semgrep org in bulk without adding or changing your CI workflows.
+
+This is an alternative method to [adding Semgrep in CI](/deployment/add-semgrep-to-ci). Instead of adding a Semgrep job or workflow to your CI/CD pipeline, repositories are added to Semgrep AppSec Platform.
+
+## Feature maturity and support
+
+- Cloud scanning is in **public beta** for existing Semgrep customers on a paid plan.
+- Cloud scanning supports hosted GitHub (GitHub.com) and GitHub Enterprise Server plans.
+    - This guide provides self-service enablement steps for **hosted GitHub plans**.
+    - For GitHub Enterprise Server users, contact your technical account manager (TAM).
+- Please leave feedback by either contacting your technical account manager (TAM) or through the **<i class="fa-solid fa-bullhorn"></i> Feedback** form in Semgrep AppSec Platform's navigation bar.
+
+## Requirements
+
+Cloud scanning requires **[<i class="fas fa-external-link fa-xs"></i> read access](https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28)** to your code in GitHub for the repositories you choose to scan. Semgrep clones your repository at the beginning of every scan. Once the scan completes, the clone is destroyed and is not persisted anywhere.
```

Comment:
@jsmoxon Hey Jack! I used the link provided in internal docs, however, it's a list of endpoints - is this intentional? Do we have a more informative link?

Reply:
Good question. I see 2 options (open to suggestions):

1. We could link to the GitHub docs on GitHub Apps generally. https://docs.github.com/en/apps/using-github-apps/authorizing-github-apps

2. Or we could link directly to the current link which I believe explains which permissions are granted via the App. I found the current link by following from the GitHub Apps docs and leading towards where they guide you to learn more about permissions. 

```For more information about what different permissions enable a GitHub App to do, see "[Choosing permissions for a GitHub App](https://docs.github.com/en/apps/creating-github-apps/setting-up-a-github-app/choosing-permissions-for-a-github-app).```

@zyannes What do you think makes the most sense?

Reply:
Hi. My issue with the current link is that it's a list of **possible** permissions we could have for our app, not the actual permission we ask for. Is it **[repository permissions for projects](https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-projects)** or is it **[Organization permissions for projects](https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#organization-permissions-for-projects)**?

We could also just not have a link. WDYT?

Reply:
Resolved with link to repository permissions.